fpga_interchange/site_router_tests/common/run_script.tpl.py

```diff
@@ -0,0 +1,31 @@
+import sys
```

Comment:
Why do you need this template that generates Python? Would there be a way for this to be a regular script with runtime command line arguments instead? It would be easier to edit that way, for a start.

Reply:
Unsure it the script specified in --run can take arguments. I mean in: nextpnr-fpga_interchange --run fpga_interchange/examples/tests/lut_testcase/lut_test.py

Reply:
Oh right, the solution here is to use an environment variable

Reply:
Sure, we can use variables. Changed that.